plugins/person-extras/src/components/SkillsEditor.svelte

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright © 2020, 2021 Anticrm Platform Contributors.
+
+Licensed under the Eclipse Public License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may
+obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script lang="ts">
+  import workbench from '@anticrm/workbench/src/__model__'
+
+  import Icon from '@anticrm/platform-ui/src/components/Icon.svelte'
+  import SkillEditor from './SkillEditor.svelte'
+  import personExtras, { Skill } from '..'
+
+  export let skills: Skill[]
+
+  function addSkill() {
+    skills = [
+      ...skills,
+      {
+        __embedded: true,
+        _class: personExtras.class.Skill,
+        level: 0,
+        skill: ''
+      }
+    ]
+  }
+</script>
+
+{#if skills.length > 0}
+  <div class="root">
+    {#each skills as skill}
```

Comment:
Why not SkillsEditor, will we be able to remove previously added skills?

Reply:
Not sure I understand the question, ideally we should introduce `ListEditor` component that can work with array, render necessary component for item and perform all necessary actions (add, remove, maybe swap), but in general I think we will get rid of these components and replace them with properties form #330